### ISSUE TITLE
Is this also providing authentication like the Zimbra Drive extension?

### ISSUE THREAD

**Issue by Priyantha** [julien-nc/integration_zimbra#7]
Hi,

I am currently running a (varly old) Nextcloud version with the 'Zimbra Drive' extension, this mostly because this creates a sync between the Zimbra users and Nextcloud so I don't need to create 'local' users for Nextcloud. Everybody who has a Zimbra account can login en view their own files.

If I would upgrade Nextcloud, and remove that old 'Zimbra Drive' extension and replace it with this new one, will I get the same result?

**Comment by barrydegraaff** [comment]
No, but you should remove the old integration (app in Nextcloud) as that is not working on newer versions of Nextcloud.

You can set-up Nextcloud to use Zimbra LDAP, to do the same.

Similar to https://github.com/Zimbra-Community/owncloud-zimlet/blob/soapServiceBarry/wiki/Connect%20ownCloud%20to%20Zimbra%20LDAP%20for%20accounts%20and%20credentials.md
